web/blueprints/user/__init__.py

```diff
@@ -1272,3 +1273,33 @@ def resend_confirmation_mail():
         session.session.commit()
 
         return redirect(return_url)
+
+
+@nav.navigate('Archivable users')
+@bp.route('/archivable_users')
+def archivable_users():
+    table = ArchivableMembersTable(data_url=url_for('.archivable_users_json'))
+    return render_template('user/archivable_users.html', table=table)
+
+
+@bp.route('/archivable_users_table')
+def archivable_users_json():
+    T = ArchivableMembersTable
+    return {'items': [
+        T.row(
+            id=info.User.id,
+            user=T.user.value(
+                title=info.User.name,
+                href=url_for('user.user_show', user_id=info.User.id)
+            ),
+            room_shortname=info.User.room and T.room_shortname.value(
+                title=info.User.room.short_name,
+                href=url_for('facilities.room_show', room_id=info.User.room.id)
+            ),
+            current_properties=" ".join(("~" if p.denied else "") + p.property_name
+                                        for p in info.User.current_properties_maybe_denied),
+            num_hosts=len(info.User.hosts),
+            # TODO better: `DateColumn.value`
+            end_of_membership=date_format(info.mem_end.date())
+        ) for info in get_archivable_members()
+    ]}
```

Comment:
@ibot3 I would like feedback in particular about this strongly-typed way of constructing the table's columns. At least PyCharm was able to autocomplete the kwargs of `.value()` and `.row()`.
See #52 .